src/components/SignUp.js

```diff
@@ -85,15 +89,40 @@ export default function SignUp() {
       // Send the POST request to backend endpoint
       const response = await http.post('/auth/signup', signUpDto);
       console.log(response.data);
+      //window.location.reload();
+      setSuccessMessage("User Registered Successfully!");
+      setErrorMessage("");
 
     // Catch username already exists error.
     } catch (error) {
         console.error('Error signing up:', error.response.data.message);
         setErrorMessage(error.response.data.message);
+        setSuccessMessage("");
 
     }
   };
 
+  const successDiv =successMessage
+      ? <Alert
+          action={
+
+            <Button color="inherit" size="small"
+                    endIcon={<ExitToAppOutlinedIcon />}
+                    aria-label ="Click to Login"
+                    onClick={() => {
+                      navigate("/sign-in");
+                    }}>
+              Login
+            </Button>
+          }
+      >{successMessage}</Alert>
+      : '';
```

Comment:
This is redundant. A better way to do this is: 

```suggestion
const successDiv = successMessage && (
   <Alert action={...} >
    {successMessage}
   </Alert>
  )
```

Reply:
And I would put it inline.

Reply:
These are conditionally appeared, If I changed it the alert section is always displayed, 
![image](https://github.com/CMSC-495-Group-One/TaskManagementUI/assets/60456566/f14af596-ccb9-44cf-a3f4-8ba28388ef4c)

Reply:
You can conditionally render the entire div So that doesn't happen